draft-ietf-quic-recovery.md

```diff
@@ -522,9 +524,14 @@ removed from bytes in flight when the Initial and Handshake keys are discarded.
 ### Sending Probe Packets
 
 When a PTO timer expires, a sender MUST send at least one ack-eliciting packet
-as a probe, unless there is no data available to send.  An endpoint MAY send up
-to two full-sized datagrams containing ack-eliciting packets, to avoid an
-expensive consecutive PTO expiration due to a single lost datagram.
+in the packet number space as a probe, unless there is no data available to
+send.  An endpoint MAY send up to two full-sized datagrams containing
+ack-eliciting packets, to avoid an expensive consecutive PTO expiration due
```

Comment:
I think that we can drop the "containing ack-eliciting packets" here, it's redundant with the first sentence.

Reply:
It's mostly redundant, but if there are two packets, this makes it clear they have to be ack-eliciting.  This didn't change as part of this PR, FWIW.